yorkie/rpc/broadcast.go

```diff
@@ -0,0 +1,35 @@
+package rpc
+
+import (
+	"context"
+
+	"github.com/yorkie-team/yorkie/api"
+	"github.com/yorkie-team/yorkie/api/converter"
+	"github.com/yorkie-team/yorkie/internal/log"
+	"github.com/yorkie-team/yorkie/pkg/document/time"
+)
+
+// Publish publishes the given event to the given Topic.
+func (s *Server) Publish(_ context.Context, request *api.PublishRequest) (*api.PublishResponse, error) {
```

Comment:
Is there any reason to break the context chain?

Reply:
The Publish API simply publishes the requested event to the local clients, so I couldn't think of anything to use the context for right now.
Can you suggest a situation that I might consider?

Reply:
I thought it was related to the routine below, but it's not.

Please check whether the context of the routine below can be linked with the callers.
https://github.com/yorkie-team/yorkie/pull/189/files#diff-12847ffa877b49cdead390469adee37ad5b1bf871f09d554bda4403391a86443R113